main.go

```diff
@@ -198,6 +199,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&restConfigBurst, "kube-api-burst", 30,
 		"Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30")
 
+	fs.DurationVar(&drainKubeTimeout, "drain-api-timeout", time.Second*10,
+		"The maximum length of time to wait for a response before giving up on kubernetes to drain a node. A value of zero means no timeout.")
```

Comment:
```suggestion
		"The timeout of the client used for draining nodes. A value of zero means no timeout.")
```
Not sure what the ideal description is. The previous one sounded very much like the NodeDrainTimeout field

Reply:
Infinite timeout would block an entire reconcile worker. So we probably shouldn't allow 0

Reply:
Updated to not allow timeouts <= 0.